doc/releases/changelog-dev.md

```diff
@@ -4,7 +4,151 @@
+
+<h4>Exponentiate Hamiltonians with flexible Trotter products 🐖</h4>
+
+* Higher-order Trotter-Suzuki methods are now easily accessible through a new operation
+  called `TrotterProduct`.
+  [(#4661)](https://github.com/PennyLaneAI/pennylane/pull/4661)
+
+  Trotterization techniques are an affective route towards accurate and efficient
+  Hamiltonian simulation. The Suzuki-Trotter product formula allows for the ability
+  to express higher-order approximations to the matrix exponential of a Hamiltonian, 
+  and it is now available to use in PennyLane via the `TrotterProduct` operation. 
+  Simply specify the `order` of the approximation and the evolution `time`.
+
+  ```python
+  coeffs = [0.25, 0.75]
+  ops = [qml.PauliX(0), qml.PauliZ(0)]
+  H = qml.dot(coeffs, ops)
+
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.qnode(dev)
+  def circuit():
+      qml.Hadamard(0)
+      qml.TrotterProduct(H, time=2.4, order=2)
+      return qml.state()
+  ```
+
+  ```pycon
+  >>> circuit()
+  [-0.13259524+0.59790098j  0.        +0.j         -0.13259524-0.77932754j  0.        +0.j        ]
+  ```
+
+  The already-available `ApproxTimeEvolution` operation represents the special case of `order=1`.
+  It is recommended to switch over to use of `TrotterProduct` because `ApproxTimeEvolution` will be
+  deprecated and removed in upcoming releases.
+
+* TODO QDrift
+
+<h4>Building blocks for quantum phase estimation 🧱</h4>
+
+* TODO
+
+<h4>New device capabilities, integration with Catalyst, and more! ⚗️</h4>
+
+* `default.qubit` now uses the new `qml.devices.Device` API and functionality in
+  `qml.devices.qubit`. If you experience any issues with the updated `default.qubit`, please let us
+  know by [posting an issue](https://github.com/PennyLaneAI/pennylane/issues/new/choose). 
+  The old version of the device is still
+  accessible by the short name `default.qubit.legacy`, or directly via `qml.devices.DefaultQubitLegacy`.
+  [(#4594)](https://github.com/PennyLaneAI/pennylane/pull/4594)
+  [(#4436)](https://github.com/PennyLaneAI/pennylane/pull/4436)
+  [(#4620)](https://github.com/PennyLaneAI/pennylane/pull/4620)
+  [(#4632)](https://github.com/PennyLaneAI/pennylane/pull/4632)
+
+  This changeover has a number of benefits for `default.qubit`, including:
+
+  * The number of wires is now optional - simply having `qml.device("default.qubit")` is valid! If
+    wires are not provided at instantiation, the device automatically infers the required number of
+    wires for each circuit provided for execution.
+
+    ```python
+    dev = qml.device("default.qubit")
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.PauliZ("new")
+        qml.RZ(0.1, wires="device")
+        qml.Hadamard("api")
+        return qml.state()
+    ```
+
+    ```pycon
+    >>> print(qml.draw(circuit)())
+       new: ──Z────────┤  State
+    device: ──RZ(0.10)─┤  State
+       api: ──H────────┤  State
```

Comment:
🤠 

a little on the nose but lemme know what you think haha

Reply:
I think it's fun but worry it detracts from the point we're trying to make. E.g., does it now look like something about custom wire labels?